packages/client/components/NewJiraIssueInput.tsx

```diff
@@ -245,11 +245,11 @@ export default createFragmentContainer(NewJiraIssueInput, {
       }
       teamMember(teamId: $teamId) {
         ... on TeamMember {
-          suggestedIntegrations {
-            ...NewJiraIssueMenu_suggestedIntegrations
+          repoIntegrations {
+            ...NewJiraIssueMenu_repoIntegrations
             items {
-              ... on SuggestedIntegrationJira {
-                projectKey
+              ... on JiraRemoteProject {
+                key
```

Comment:
What's the reasoning behind using `key` on `JiraRemoteProject` and `projectKey` on `JiraIssue`? Perhaps `key` could be changed to `projectKey` to be more consistent?

Reply:
good find!
reason being is the jira API return a project like `{key: 'DEV3}`, so i tried to match our project to the jira API.
I'd equate to to a foreign key.  for example, `Task.teamId` === `Team.id` (just replace the word id with key)